templates/web/oxfordshire/front/_list-entry.html

```diff
@@ -0,0 +1,4 @@
+[%
+report = INCLUDE 'report/_item.html' no_fixed = 1;
+report.replace('item-list--reports__item', 'item-list--reports__item item-list__item--with-pin');
```

Comment:
Oh dear :wink: Would it be better or worse to pass in the extra class as a variable (or even a `with_pin = 1` switch) into the template, dya think? Might make the intention a bit clearer.